src/components/ciConfig/CIConfigDiffView.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect } from 'react'
+import { noop, Progressing, VisibleModal } from '../common'
+import { ReactComponent as CloseIcon } from '../../assets/icons/ic-cross.svg'
+import { Workflow } from '../workflowEditor/Workflow'
+import { Link, useHistory, useLocation, useParams, useRouteMatch } from 'react-router-dom'
+import { URLS } from '../../config'
+
+export default function CIConfigDiffView({
+    ciConfig,
+    configOverridenPipelines,
+    configOverrides,
+    processedWorkflows,
+    toggleConfigOverrideDiffModal,
+}) {
+    const history = useHistory()
+    const location = useLocation()
+    const match = useRouteMatch<{
+        appId: string
+    }>()
+    const { appId } = useParams<{
+        appId: string
+    }>()
+    const wfCIMap = new Map<number, number>()
+    const _configOverridenWorkflows = configOverrides?.workflows?.filter((_cwf) => {
```

Comment:
should we pass default value in case `configOverrides` is `undefined`?

Reply:
Changed the impl a bit. Now it's being initiated with an empty array.